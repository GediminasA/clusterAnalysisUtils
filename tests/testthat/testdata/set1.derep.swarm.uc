C	0	67	*	*	*	*	*	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925	*
S	0	18456	*	*	*	*	*	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925	*
H	0	18456	100.0	+	0	0	18456M	30161|M14_240029|UK_[England]|07/2013-06/2014;size=20	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	51145|B003|UK_[England]|07/2015-06/2016;size=15	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	37748|M14_240604|UK_[England]|07/2014-06/2015;size=11	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	47238|2161506|The_Netherlands|;size=4	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	57584|NEIS-16-579|Sweden|;size=4	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	47287|M16_240542|UK_[England]|07/2016-06/2017;size=3	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	53058|M16_240712|UK_[England]|07/2016-06/2017;size=3	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	60808|M17_240507|UK_[England]|07/2017-06/2018;size=3	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	44664|2151341|The_Netherlands|;size=2	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	58082|2172302|The_Netherlands|;size=2	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	61511|2181594_I|The_Netherlands|;size=2	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	82452|Nmen197|Spain|;size=2	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	85833|SMG-19-460|UK_[Scotland]|07/2018-06/2019;size=2	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	118886|2210261|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	35786|M14_240468|UK_[England]|07/2014-06/2015;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	37940|M15_240193|UK_[England]|07/2014-06/2015;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	37942|M15_240195|UK_[England]|07/2014-06/2015;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	38115|M15_240576|UK_[England]|07/2014-06/2015;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	38611|15-229|Sweden|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	39368|M15_240740|UK_[England]|07/2015-06/2016;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	44483|OX40482|UK_[England]|07/2014-06/2015;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	44679|2160364|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	47244|2132650|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	49307|LNP28796at|France|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	51250|B197|UK_[England]|07/2015-06/2016;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	51622|22282|Spain|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	53060|M16_240715|UK_[England]|07/2016-06/2017;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	55106|2170151|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	55114|2170769|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	55118|2171002|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	55119|2171062|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	55120|2171087|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	55562|IE16Nm62|Ireland|07/2016-06/2017;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	60549|M17_240292|UK_[England]|07/2016-06/2017;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	60929|M17_240790|UK_[England]|07/2017-06/2018;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	61468|2181316|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	63603|M19_240240|UK_[England]|07/2018-06/2019;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	72250|Nmen141|Spain|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	72308|M19_240167|UK_[England]|07/2018-06/2019;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	76562|DE14636|Germany|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	82031|SMG-20-1319|UK_[Scotland]|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	82450|Nmen195|Spain|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	83989|2182099|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	84266|DE14257|Germany|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	85539|M18_240145|UK_[Wales]|07/2017-06/2018;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	85586|M18_240332|UK_[England]|07/2017-06/2018;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	86971|2190149|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	86988|2190452_II|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	87070|SMG-19-604|UK_[Scotland]|07/2018-06/2019;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	91546|SMG-19-1710|UK_[Scotland]|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	91759|M15_240951|UK_[England]|07/2015-06/2016;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	93908|IE17Nm48|Ireland|07/2017-06/2018;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	41558|M15_240995|UK_[England]|07/2015-06/2016;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	58065|2171897|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	93241|SMG-19-1555|UK_[Scotland]|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	44660|2150367|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	45360|2160974|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	47265|M16_240516|UK_[England]|07/2015-06/2016;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	51624|22328|Spain|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	55101|2162200|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	59777|2180842|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	60589|M17_240304|UK_[England]|07/2016-06/2017;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	60803|M18_240015|UK_[England]|07/2017-06/2018;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	83985|2182022|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	91817|2191318|The_Netherlands|;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
H	0	18456	100.0	+	0	0	18456M	94286|M18_240516|UK_[England]|07/2018-06/2019;size=1	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925
C	1	69	*	*	*	*	*	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831	*
S	1	18438	*	*	*	*	*	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831	*
H	1	18438	100.0	+	0	0	18438M	20204|M11_240043|UK_[England]|07/2010-06/2011;size=20	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	37914|M15_240157|UK_[England]|07/2014-06/2015;size=7	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	37944|M15_240199|UK_[England]|07/2014-06/2015;size=7	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	42426|15-405|Sweden|;size=6	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	42527|M16_240067|UK_[Wales]|07/2015-06/2016;size=5	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	51229|B119|UK_[England]|07/2015-06/2016;size=4	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	58174|17-654|Sweden|;size=4	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	26240|1259000345|Sweden|;size=3	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	115706|BT39929V1C|UK|;size=2	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	47315|M16_240641|UK_[England]|07/2016-06/2017;size=2	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	52566|CA40856|UK_[Wales]|07/2014-06/2015;size=2	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	53218|M17_240016|UK_[Wales]|07/2016-06/2017;size=2	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	60981|M17_240800|UK_[England]|07/2017-06/2018;size=2	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	120816|BT22280V3C|UK_[England]|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	19970|M10_240520|UK_[England]|07/2010-06/2011;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	20032|M10_240632|UK_[England]|07/2010-06/2011;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	20224|M11_240065|UK_[England]|07/2010-06/2011;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	21153|M11_240710|UK_[England]|07/2011-06/2012;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	21205|M11_240801|UK_[England]|07/2011-06/2012;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	21447|M12_240241|UK_[England]|07/2011-06/2012;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	26163|1059000129|Sweden|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	26193|1159000177|Sweden|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	26448|R148|UK|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	27545|20374|UK_[England]|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18439	100.0	+	0	0	16375MI2063M	30672|IBD-514|Norway|07/2011-06/2012;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	34503|M13_240485|UK_[England]|07/2013-06/2014;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	35376|12.8408.J|UK_[Scotland]|07/2012-06/2013;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	35567|M14_240056|UK_[England]|07/2013-06/2014;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	35804|M14_240489|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	37853|M15_240077|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	38028|M15_240319|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	38104|M15_240525|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	40287|12-539|Sweden|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	41216|LNP28392|France|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	41749|LNP27919at|France|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	42459|M16_240128|UK|07/2015-06/2016;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	44420|ST40264|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	44564|OX41493|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	44748|M16_240280|UK_[England]|07/2015-06/2016;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	47283|M16_240538|UK_[England]|07/2016-06/2017;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	49508|CA41506|UK_[Wales]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	49757|CM40605|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	49789|CM40919|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	50182|WG41112|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	50233|PL41608|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	52552|BR41445|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	53122|M16_240797|UK_[England]|07/2016-06/2017;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	59581|SMG_18_431|UK_[Scotland]|07/2017-06/2018;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	60968|M18_240046|UK_[England]|07/2017-06/2018;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	61494|2181448_1|The_Netherlands|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	74305|20-247|Sweden|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	80158|M20_240083|UK_[England]|07/2019-06/2020;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	85941|SMG-19-526|UK_[Scotland]|07/2018-06/2019;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	86982|2190423|The_Netherlands|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	50251|BR40257|UK_[England]|07/2014-06/2015;size=2	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	27543|22238|UK_[England]|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	53330|M17_240169|UK_[England]|07/2016-06/2017;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	61013|M18_240058|UK_[England]|07/2017-06/2018;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	35424|M13_240529|UK_[England]|07/2013-06/2014;size=2	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	122598|SMG-22-1883|UK_[Scotland]|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	26214|1259000018|Sweden|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	27501|NO01021203|UK_[England]|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	38116|M15_240600|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	49378|CA40036|UK_[Wales]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	49700|LO40812|UK_[England]|07/2014-06/2015;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	61414|2172559|The_Netherlands|;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	80153|M20_240068|UK_[England]|07/2019-06/2020;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
H	1	18438	100.0	+	0	0	18438M	44760|M16_240362|UK_[England]|07/2015-06/2016;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831
