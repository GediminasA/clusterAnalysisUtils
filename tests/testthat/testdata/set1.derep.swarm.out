swarm_1	2	28134|M13_240015|UK_[England]|07/2012-06/2013;size=925,30161|M14_240029|UK_[England]|07/2013-06/2014;size=20,51145|B003|UK_[England]|07/2015-06/2016;size=15,37748|M14_240604|UK_[England]|07/2014-06/2015;size=11,47238|2161506|The_Netherlands|;size=4,57584|NEIS-16-579|Sweden|;size=4,47287|M16_240542|UK_[England]|07/2016-06/2017;size=3,53058|M16_240712|UK_[England]|07/2016-06/2017;size=3,60808|M17_240507|UK_[England]|07/2017-06/2018;size=3,44664|2151341|The_Netherlands|;size=2,58082|2172302|The_Netherlands|;size=2,61511|2181594_I|The_Netherlands|;size=2,82452|Nmen197|Spain|;size=2,85833|SMG-19-460|UK_[Scotland]|07/2018-06/2019;size=2,118886|2210261|The_Netherlands|;size=1,35786|M14_240468|UK_[England]|07/2014-06/2015;size=1,37940|M15_240193|UK_[England]|07/2014-06/2015;size=1,37942|M15_240195|UK_[England]|07/2014-06/2015;size=1,38115|M15_240576|UK_[England]|07/2014-06/2015;size=1,38611|15-229|Sweden|;size=1,39368|M15_240740|UK_[England]|07/2015-06/2016;size=1,44483|OX40482|UK_[England]|07/2014-06/2015;size=1,44679|2160364|The_Netherlands|;size=1,47244|2132650|The_Netherlands|;size=1,49307|LNP28796at|France|;size=1,51250|B197|UK_[England]|07/2015-06/2016;size=1,51622|22282|Spain|;size=1,53060|M16_240715|UK_[England]|07/2016-06/2017;size=1,55106|2170151|The_Netherlands|;size=1,55114|2170769|The_Netherlands|;size=1,55118|2171002|The_Netherlands|;size=1,55119|2171062|The_Netherlands|;size=1,55120|2171087|The_Netherlands|;size=1,55562|IE16Nm62|Ireland|07/2016-06/2017;size=1,60549|M17_240292|UK_[England]|07/2016-06/2017;size=1,60929|M17_240790|UK_[England]|07/2017-06/2018;size=1,61468|2181316|The_Netherlands|;size=1,63603|M19_240240|UK_[England]|07/2018-06/2019;size=1,72250|Nmen141|Spain|;size=1,72308|M19_240167|UK_[England]|07/2018-06/2019;size=1,76562|DE14636|Germany|;size=1,82031|SMG-20-1319|UK_[Scotland]|;size=1,82450|Nmen195|Spain|;size=1,83989|2182099|The_Netherlands|;size=1,84266|DE14257|Germany|;size=1,85539|M18_240145|UK_[Wales]|07/2017-06/2018;size=1,85586|M18_240332|UK_[England]|07/2017-06/2018;size=1,86971|2190149|The_Netherlands|;size=1,86988|2190452_II|The_Netherlands|;size=1,87070|SMG-19-604|UK_[Scotland]|07/2018-06/2019;size=1,91546|SMG-19-1710|UK_[Scotland]|;size=1,91759|M15_240951|UK_[England]|07/2015-06/2016;size=1,93908|IE17Nm48|Ireland|07/2017-06/2018;size=1,41558|M15_240995|UK_[England]|07/2015-06/2016;size=1,58065|2171897|The_Netherlands|;size=1,93241|SMG-19-1555|UK_[Scotland]|;size=1,44660|2150367|The_Netherlands|;size=1,45360|2160974|The_Netherlands|;size=1,47265|M16_240516|UK_[England]|07/2015-06/2016;size=1,51624|22328|Spain|;size=1,55101|2162200|The_Netherlands|;size=1,59777|2180842|The_Netherlands|;size=1,60589|M17_240304|UK_[England]|07/2016-06/2017;size=1,60803|M18_240015|UK_[England]|07/2017-06/2018;size=1,83985|2182022|The_Netherlands|;size=1,91817|2191318|The_Netherlands|;size=1,94286|M18_240516|UK_[England]|07/2018-06/2019;size=1	19977|M10_240531|UK_[England]|07/2010-06/2011;size=831,20204|M11_240043|UK_[England]|07/2010-06/2011;size=20,37914|M15_240157|UK_[England]|07/2014-06/2015;size=7,37944|M15_240199|UK_[England]|07/2014-06/2015;size=7,42426|15-405|Sweden|;size=6,42527|M16_240067|UK_[Wales]|07/2015-06/2016;size=5,51229|B119|UK_[England]|07/2015-06/2016;size=4,58174|17-654|Sweden|;size=4,26240|1259000345|Sweden|;size=3,115706|BT39929V1C|UK|;size=2,47315|M16_240641|UK_[England]|07/2016-06/2017;size=2,52566|CA40856|UK_[Wales]|07/2014-06/2015;size=2,53218|M17_240016|UK_[Wales]|07/2016-06/2017;size=2,60981|M17_240800|UK_[England]|07/2017-06/2018;size=2,120816|BT22280V3C|UK_[England]|;size=1,19970|M10_240520|UK_[England]|07/2010-06/2011;size=1,20032|M10_240632|UK_[England]|07/2010-06/2011;size=1,20224|M11_240065|UK_[England]|07/2010-06/2011;size=1,21153|M11_240710|UK_[England]|07/2011-06/2012;size=1,21205|M11_240801|UK_[England]|07/2011-06/2012;size=1,21447|M12_240241|UK_[England]|07/2011-06/2012;size=1,26163|1059000129|Sweden|;size=1,26193|1159000177|Sweden|;size=1,26448|R148|UK|;size=1,27545|20374|UK_[England]|;size=1,30672|IBD-514|Norway|07/2011-06/2012;size=1,34503|M13_240485|UK_[England]|07/2013-06/2014;size=1,35376|12.8408.J|UK_[Scotland]|07/2012-06/2013;size=1,35567|M14_240056|UK_[England]|07/2013-06/2014;size=1,35804|M14_240489|UK_[England]|07/2014-06/2015;size=1,37853|M15_240077|UK_[England]|07/2014-06/2015;size=1,38028|M15_240319|UK_[England]|07/2014-06/2015;size=1,38104|M15_240525|UK_[England]|07/2014-06/2015;size=1,40287|12-539|Sweden|;size=1,41216|LNP28392|France|;size=1,41749|LNP27919at|France|;size=1,42459|M16_240128|UK|07/2015-06/2016;size=1,44420|ST40264|UK_[England]|07/2014-06/2015;size=1,44564|OX41493|UK_[England]|07/2014-06/2015;size=1,44748|M16_240280|UK_[England]|07/2015-06/2016;size=1,47283|M16_240538|UK_[England]|07/2016-06/2017;size=1,49508|CA41506|UK_[Wales]|07/2014-06/2015;size=1,49757|CM40605|UK_[England]|07/2014-06/2015;size=1,49789|CM40919|UK_[England]|07/2014-06/2015;size=1,50182|WG41112|UK_[England]|07/2014-06/2015;size=1,50233|PL41608|UK_[England]|07/2014-06/2015;size=1,52552|BR41445|UK_[England]|07/2014-06/2015;size=1,53122|M16_240797|UK_[England]|07/2016-06/2017;size=1,59581|SMG_18_431|UK_[Scotland]|07/2017-06/2018;size=1,60968|M18_240046|UK_[England]|07/2017-06/2018;size=1,61494|2181448_1|The_Netherlands|;size=1,74305|20-247|Sweden|;size=1,80158|M20_240083|UK_[England]|07/2019-06/2020;size=1,85941|SMG-19-526|UK_[Scotland]|07/2018-06/2019;size=1,86982|2190423|The_Netherlands|;size=1,50251|BR40257|UK_[England]|07/2014-06/2015;size=2,27543|22238|UK_[England]|;size=1,53330|M17_240169|UK_[England]|07/2016-06/2017;size=1,61013|M18_240058|UK_[England]|07/2017-06/2018;size=1,35424|M13_240529|UK_[England]|07/2013-06/2014;size=2,122598|SMG-22-1883|UK_[Scotland]|;size=1,26214|1259000018|Sweden|;size=1,27501|NO01021203|UK_[England]|;size=1,38116|M15_240600|UK_[England]|07/2014-06/2015;size=1,49378|CA40036|UK_[Wales]|07/2014-06/2015;size=1,49700|LO40812|UK_[England]|07/2014-06/2015;size=1,61414|2172559|The_Netherlands|;size=1,80153|M20_240068|UK_[England]|07/2019-06/2020;size=1,44760|M16_240362|UK_[England]|07/2015-06/2016;size=1
